src/v2/view-builder/views/device/DeviceChallengePollView.js

```diff
@@ -145,6 +147,23 @@ const Body = BaseFormWithPolling.extend(
       });
     },
 
+    showCustomFormErrorCallout(error) {
+      let options = {
```

Comment:
nit: Should be able to treat this as a `const` as we're only modifying object props.

Reply:
updated